frontend/public/components/nav/__tests__/perspective-nav.spec.tsx

```diff
@@ -0,0 +1,149 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import { usePinnedResources } from '@console/shared';
+import { useActivePerspective } from '@console/dynamic-plugin-sdk';
+import { useExtensions } from '@console/plugin-sdk';
+import { mockPerspectiveExtensions } from '@console/dynamic-plugin-sdk/src/perspective/__tests__/perspective.data';
+import { DragAndDrop } from '@console/shared/src/components/dnd';
+import { NavGroup } from '@patternfly/react-core';
+import PerspectiveNav from '../perspective-nav';
+import AdminNav from '../admin-nav';
+import PinnedResource from '../PinnedResource';
+const useActivePerspectiveMock = useActivePerspective as jest.Mock;
```

Comment:
a line break would be good here, between import and variable declarations

```suggestion
import PinnedResource from '../PinnedResource';

const useActivePerspectiveMock = useActivePerspective as jest.Mock;
```